src/utils/checkAccountExistence.ts

```
@@ -0,0 +1,15 @@
+import { Server } from "stellar-sdk";
+
+const checkAccountExistence = async (accountId: string) => {
+  try {
+    const server = new Server("https://horizon-testnet.stellar.org");
+
+    const account = await server.accounts().accountId(accountId).call();
+
+    return account ? true : false;
+  } catch {
+    return false;
```

We should throw an error here.